src/utils/format/formatDifficulty.ts

```diff
@@ -0,0 +1,16 @@
+const DIFFICULTIES = {
+  EASY: 'Fácil',
+  MEDIUM: 'Media',
+  HARD: 'Difícil',
+};
+
+export function formatDifficulty(difficulty: 1 | 2 | 3) {
```

Comment:
The backend updated in one of the routes the information they provided me for difficulty. Instead of returning the id, they started returning the actual value. I am not sure if they will update this for you too or if this is expected to be the final version

Reply:
I think for this functionality, this is the final difficulty format